python/graphstorm/dataloading/dataset.py

```diff
@@ -155,29 +155,29 @@ def prepare_batch_edge_input(g, input_edges,
     return feat
 
 class GSgnnData():
-    """ The GraphStorm data
+    """ The GraphStorm data class.
 
     Parameters
     ----------
     part_config : str
-        The path of the partition configuration file.
+        The path of the partition configuration JSON file.
     node_feat_field: str or dict of list of str
-        The fields of the node features that will be encoded by GSNodeInputLayer.
+        The fields of the node features that will be encoded by ``GSNodeInputLayer``.
         It's a dict if different node types have
         different feature names.
-        Default: None
+        Default: None.
     edge_feat_field : str or dict of list of str
-        The fields of the edge features.
+        The fields of the edge features to be encoded by later released ``GSEdgeInputLayer``.
```

Comment:
what does later released mean?

Reply:
updated with `This argument is reserved for future usage when the ``GSEdgeInputLayer`` is implemented.`